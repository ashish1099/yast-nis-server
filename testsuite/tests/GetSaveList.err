2001-07-04 11:54:17 <3> bia(31943) nis_server/io.ycp:488 Cannot evaluate unknown term tooboolean (true)
2001-07-04 11:54:17 <3> bia(31943) nis_server/io.ycp:488 Logical operator && not allowed with non boolean argument nil
2001-07-04 11:54:17 <3> bia(31943) nis_server/io.ycp:488 condition (! (tooboolean (lookup (user_settings, ".var.yp.makefile.NOPUSH", true))) && (size (lookup (user_settings, ".var.yp.ypservers", [])) > 0)) in if statement evaluates to nil, but only true and false are allowed
