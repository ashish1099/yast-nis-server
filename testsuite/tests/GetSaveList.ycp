/*
 * YaST2: Modules testsuite
 *
 * Description:
 *   Testsuite for the GetSaveList function
 *
 * Authors:
 *   Dan Vesely <dan@suse.cz>
 *
 * $Id$
 */

{
    include "testsuite.ycp";
    include "nis_server/io.ycp";
    include "nis_server/routines.ycp";

    map user_settings = $[];

    TEST (``(GetSaveList (`none)), [], true);
    TEST (``(GetSaveList (`slave)), [], true);
    TEST (``(GetSaveList (`master)), [], true);

}
