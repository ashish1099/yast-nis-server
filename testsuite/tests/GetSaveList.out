Return	[$["function":YPDaemons (`none), "progress_label":"Stoping running daemons.", "save_label":"Stop running daemons."], $["function":SaveVariables (`none), "progress_label":"Setting config variables.", "save_label":"Set config variables."]]
Return	[$["function":YPGetMaps (), "progress_label":"Gettins maps from master.", "save_label":"Get maps from master."], $["function":SaveVariables (`slave), "progress_label":"Setting config variables.", "save_label":"Set config variables."], $["function":SaveSecurenets (), "progress_label":"Saving hosts allowed to query server.", "save_label":"Save hosts allowed to query server."], $["function":YPDaemons (`slave), "progress_label":"Starting daemon.", "save_label":"Start daemon."]]
Return	[$["function":SaveVariables (`master), "progress_label":"Setting config variables.", "save_label":"Set config variables."], $["function":SaveSecurenets (), "progress_label":"Saving hosts allowed to query server.", "save_label":"Save hosts allowed to query server."], $["function":YPDaemons (`master), "progress_label":"Starting daemons.", "save_label":"Start daemons."], $["function":YPMake (), "progress_label":"Creating database.", "save_label":"Create initial database."]]
